packages/qwik-image/src/lib/image.tsx

```diff
@@ -226,9 +229,14 @@ export const Image = component$<ImageProps>((props) => {
     ...getStyles(props),
   }));
   const sizes = useComputed$(() => getSizes(props));
-  const srcSet = useComputed$(() => {
-    const { src, width, height, aspectRatio, layout } = props;
-    return getSrcSet({
+  useTask$(async ({ track }) => {
+    const src = track(() => props.src);
+    const width = track(() => props.width);
+    const height = track(() => props.height);
+    const aspectRatio = track(() => props.aspectRatio);
+    const layout = track(() => props.layout);
+
+    srcSetSig.value = await getSrcSet({
```

Comment:
FYI, you can run any sync you like inside track and track multiple signals at once.

you could so something like `await getSrcSet(track(props))`

Reply:
Thanks for sharing

Reply:
actually I just realized that you need to reference the props themselves, so it would be more like `await getSrcSet(track(() => ({src: props.src, width: props.width, ...})` or you could destructure in the component top and just list the props in track.

Reply:
I see, would you like to create a PR for that?